code/modules/jobs/job_types/!crashsite/syndi_doctor.dm

```diff
@@ -0,0 +1,43 @@
+/datum/job/syndi_doctor
+	title = "Syndicate Doctor"
```

Comment:
Personally, I prefer these roles to have the other names, so that it's less "Syndicate this" "Syndicate That" and they hold up on their own.